apps/backend/src/datasources/postgres/ExperimentDataSource.ts

```diff
@@ -672,60 +691,6 @@ export default class PostgresExperimentDataSource
       });
   }
 
-  async getExperiments({ filter }: ExperimentsArgs): Promise<Experiment[]> {
```

Comment:
suggestion: since this method name is free, do you think we could use `getExperiments` instead of `getAllExperiments`. It is shorter and more consistant with other data sources like `async getCalls(filter?: CallsFilter): Promise<Call[]>`